pkg/utils/resources/resources.go

```diff
@@ -63,6 +63,23 @@ func Merge(resources ...v1.ResourceList) v1.ResourceList {
 	return result
 }
 
+// MergeResourceLimitsIntoRequests merges resource limits into requests if no request exists for the given resource
+func MergeResourceLimitsIntoRequests(container v1.Container) v1.ResourceList {
```

Comment:
A `Ceiling` method is very similar to the `RequestsForPods` method which already exists.  I opted to stick with the existing method rather than rename it to `Ceiling`.  (although we could easily rename it if we wanted to).

Reply:
I ended up implementing a Ceiling method :).  Right now it takes in a single PodSpec and returns ResourceRequirements.